docs/ref/contrib/admin/index.txt

```diff
@@ -816,6 +816,7 @@ subclass::
 
           my_property.short_description = "Full name of the person"
           my_property.admin_order_field = "last_name"
+          my_property.boolean = True
```

Comment:
If you run `make black` in the docs directory, that should fix the failing blacken docs GitHub action.
Should we put `my_property.boolean = False` here as this example of a boolean doesn't make sense? Maybe also:
```python
              @admin.display(
                  ordering="last_name",
                  description="Full name of the person",
                  boolean=False,
              )
              def full_name(self):
                  return self.first_name + " " + self.last_name
```
Might want to add a `.. versionadded:: 5.0` note (or 5.1 if we don't get it in in time) that specifically draws attention to the new boolean property and what it does

I think we maybe want a small release note within the `django.contrib.admin` section

Reply:
> I think we maybe want a small release note within the `django.contrib.admin` section

Yes, for example:
```
* Properties in :attr:`.ModelAdmin.list_display` now support ``boolean``
  attribute.